mne/viz/topomap.py

```diff
@@ -1184,7 +1184,13 @@ def plot_tfr_topomap(tfr, tmin=None, tmax=None, fmin=None, fmax=None,
     if not show_names:
         names = None
 
-    data = tfr.data
+    data = tfr.data[picks, :, :]
+
+    # merging grads before rescaling makes ERDs visible
+    if merge_grads:
+        from ..channels.layout import _merge_grad_data
+        data = _merge_grad_data(data)
+
```

Comment:
Maybe this should actually be refactored into its own function, or `_merge_grad_data` amended, something like

`data = _merge_grad_data(data, merge_grads)`

which would just pass the data unprocessed if `merge_grads == False`